Improve next_pass documentation (#9762)
godotengine/godot-docs: tutorials/3d/standard_material_3d.rst

Render priority
---------------

> This is mostly

Reviewer: Why this change? In either case this new sentence is not very clear, the "this" part is confusing IMO when nothing is referred to

Author: Ah. It was a run on sentence, originally I'd put the other one first. Will fix.

useful for transparent objects (or opaque objects that perform depth draw
but no color draw, such as cracks on the floor).

> Objects are Sorted by opaque/transparent queue, then render_priority. With higher priority being drawn later. Depth testing overrules priority. Priority alone cannot force opaque objects to be drawn over eachother.

Reviewer: ```suggestion
Objects are Sorted by opaque/transparent queue, then :ref:`render_priority<class_Material_property_render_priority>`. With higher priority being drawn later. Depth testing overrules priority. Priority alone cannot force opaque objects to be drawn over eachother.
```
Or
```suggestion
Objects are Sorted by opaque/transparent queue, then ``render_priority``. With higher priority being drawn later. Depth testing overrules priority. Priority alone cannot force opaque objects to be drawn over eachother.
```

Reviewer: Also this line needs to be broken up, it's too long

Next Pass
---------

> Setting next_pass material will cause an object to be rendered again with that next material. Sorted by opaque/transparent queue, then render_priority, then depth.

Reviewer: Same here

.. image:: img/next_pass.webp

> Depth will test equal between both materials unless the grow setting or other vertex transformations are used. Multiple transparent passes should use render_priority to ensure correct ordering.

Reviewer: And here

useful for transparent objects (or opaque objects that perform depth draw
but no color draw, such as cracks on the floor).

> Objects are Sorted by opaque/transparent queue, then :ref:`render_priority<class_Material_property_render_priority>`.

Reviewer 2: ```suggestion
Objects are sorted by opaque/transparent queue, then :ref:`render_priority<class_Material_property_render_priority>`.
```

With higher priority being drawn later.

> Depth testing overrules priority. Priority alone cannot force opaque objects to be drawn over eachother.

Reviewer: ```suggestion
Depth testing overrules priority. Priority alone cannot force opaque objects to be drawn over each other.
```

Next Pass
---------

> Setting :ref:`next_pass<class_Material_property_next_pass>` material will cause an object to be rendered again with that next material.

Reviewer 2: ```suggestion
Setting :ref:`next_pass<class_Material_property_next_pass>` on a material will cause an object to be rendered again with that next material.
```

> Sorted by opaque/transparent queue, then :ref:`render_priority<class_Material_property_render_priority>`, then depth.

Reviewer 2: ```suggestion
Objects are sorted by opaque/transparent queue, then :ref:`render_priority<class_Material_property_render_priority>`. Transparent objects are also sorted by depth.
```

useful for transparent objects (or opaque objects that perform depth draw
but no color draw, such as cracks on the floor).

> Objects are sorted by opaque/transparent queue, then :ref:`render_priority<class_Material_property_render_priority>`.

Reviewer 3: ```suggestion
Objects are sorted by an opaque/transparent queue, then :ref:`render_priority<class_Material_property_render_priority>`,
```

> With higher priority being drawn later.

Reviewer 3: ```suggestion
with higher priority being drawn later.
```
